src/HealthChecks.Gcp.CloudStorage/HealthChecks.Gcp.CloudStorage.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardTargetVersion)</TargetFramework>
+    <PackageLicenseUrl>$(PackageLicenseUrl)</PackageLicenseUrl>
```

Comment:
It makes no sense to use such an assignment. Just remove all these ```<PropXXX>$(PropXXX)</PropXXX>```. That was the whole point - move the duplicate settings to one place

Reply:
Got it, pushing now the change, although <TargetFramework>$(NetStandardTargetVersion)</TargetFramework> i dont see how to replace it since it can be netstandard2.0 or netcoreapp3.0 and this is only on my contributions projects all others needs to be changed as well

Reply:
Thanks.

I take a closer look at the project structure and I have a few suggestions.

1. Move this section from dependencies.props to Directory.Build.props because it is not related to depemdencies.
```
<PropertyGroup Label="Package information">
 ...
</PropertyGroup>
```

2. Remove all duplicated code from other project csprojs as you did for Cifs.

Reply:
for the point 1 ok agreed and will change for my 2 projects SmbCifs and CloudStorage, the ones i actually did with this pull request.

for the point 2, the change on all others i think its best to create a new PR after this one is approved and merged by the owner/s of repo!

Reply:
Actually looking again i would prefer for any of this change to be done on another PR both for th the point 1 and 2 as this is global to the full project!

Reply:
As you wish.